### ISSUE TITLE
Add kableExtra package, please

### ISSUE THREAD

**Issue by ciaranmci** [opensafely-core/r-docker#74]
I want to produce a publicaiton-ready table using the [kableExtra](https://www.rdocumentation.org/packages/kableExtra/versions/1.3.4) package in R. I previously requested installation of the [magick](https://github.com/opensafely-core/r-docker/issues/67) package because the developers of kable libraries [mentioned it as a dependency](https://haozhu233.github.io/kableExtra/save_kable_and_as_image.html)...but of course I never requested installation of kableExtra.

I haven't needed the other dependency ([webshot](https://www.rdocumentation.org/packages/webshot/versions/0.5.2/) package) to produce the publication-ready table when running a local instance of R (not in the OpenSAFELY docker). Would it be worth installing that too, or is its webpage functionality a No-No for containers?

**Comment by ciaranmci** [comment]
I've noticed that the kableExtra and webshot packages have been installed. Thank you.

My local OpenSAFELY docker container still fails to save a kable table object using the kableExtra::[save_kable()](https://www.rdocumentation.org/packages/kableExtra/versions/1.3.4/topics/save_kable).
The guidance on [feature dependency](https://haozhu233.github.io/kableExtra/save_kable_and_as_image.html) notes that the machine might need to run webshot::[install_phantomjs()](https://www.rdocumentation.org/packages/webshot/versions/0.5.2/topics/install_phantomjs):
![image](https://user-images.githubusercontent.com/44294573/159953422-3e9c7322-e2d0-4a69-80f3-de4e76c87eed.png)

Trying to call webshot::install_phantomjs() obviously throws an error in my local OpenSAFELY docker container (I assume because a call is being made to download something from outside).

**Am I correct to assume that my lovely kable tables are probably not possible if kableExtra needs PhantomJS installed in the environment?**

Log below:
![image](https://user-images.githubusercontent.com/44294573/159953052-c9548a34-c036-421a-af26-c8e6665b264e.png)

**Comment by bloodearnest** [comment]
Yes, that sounds like the case. We may be able to install it into the docker image, depending on how big it is. I'll look into it.
